docs/firestore_schema/version0.md

```diff
@@ -0,0 +1,75 @@
+# Version 0 Schema
+## Operations
+```typescript
+type Operation {
+    // Schema version
+    version: int,
+    // Has the operation been applied to the blockchain?
+    applied: boolean,
+    // ISO-formatted datetime at which this operation was included in the blockchain
+    block_at: string,
+    // Sequence number of the block cointaining this operation
+    block_seq: int,
+    // ISO-formatted datetime at which this operation document was created
+    created_at: string,
+    // MID of the creator of this operation
+    creator_mid: string,
+    // UID of the creator of this operation - auth backend specific
+    creator_uid: string,
+    // The type of this operation  - one of [REQUEST_INVITE and TRUST]
+    op_code: string,
+    // An object containing information specific to the operation type
+    data: object,
```

Comment:
Make this a union type of actual object specifications by type. Even better, couple this with the operation type, so that you can enforce by the schema that you can't have a mismatched data blob and operation.